packages/pr-files-checker/README.md

```diff
@@ -0,0 +1,46 @@
+# GitHub PR diff checker
```

Comment:
**suggestion:** update the title and description with the actual name of the action.
Also mention the permissions required by the GH_TOKEN to make things clearer for user. This may be something like in https://github.com/bonitasoft/actions/tree/main/packages/pr-title-conventional-commits#permissions